File: Modulo4-ObjectiveC-Lorena/Exercicio6/Exercicio6.m
//
//  main.m
//  Exercicio6-C
//
//  Created by Lorena Rodrigues Bruno on 10/09/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

#import <Foundation/Foundation.h>
#import "PontoCartesiano.h"
# import "Triangulo.h"

@implementation PontoCartesiano
@synthesize pontoX, pontoY;
-(id)initPonto:(int)pontoX Segundo:(int)pontoY {
    self = [super init];
    self.pontoX = pontoX;
    self.pontoY = pontoY;
    return self;
}
@end

@implementation Triangulo
-(id)initTriangulo {
    self = [super init];
    ponto1 = [[PontoCartesiano alloc]initPonto:10 Segundo:15];
    ponto2 = [[PontoCartesiano alloc]initPonto:5 Segundo:10];
    ponto3 = [[PontoCartesiano alloc]initPonto:15 Segundo:10];
    return self;
}

-(void)imprimeTriangulo {
    printf("Ponto 1: (%i, %i)\nPonto 2: (%i, %i)\nPonto 3: (%i, %i)\n", ponto1.pontoX, ponto1.pontoY, ponto2.pontoX, ponto2.pontoY, ponto3.pontoX, ponto3.pontoY);
}
@end

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        Triangulo *triangulo1 = [[Triangulo alloc]initTriangulo];
        [triangulo1 imprimeTriangulo];
    }
    return 0;
}

File: Modulo4-Lorena/Exercicio6.swift
//
//  main.swift
//  Exercicio6
//
//  Created by Lorena Rodrigues Bruno on 28/08/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

import Foundation

class Ponto {
    var eixoX = Int()
    var eixoY = Int()
    
    init(eixoX : Int, eixoY : Int) {
        self.eixoX = eixoX
        self.eixoY = eixoY
    }
    
    func imprimeCoordenada() {
        print("Coordenadas do ponto: (\(eixoX),\(eixoY)) ")
    }
    
}
class Triangulo {
    var vertice1 = Ponto(eixoX: 5, eixoY: 2)
    var vertice2 = Ponto(eixoX: 9, eixoY: 2)
    var vertice3 = Ponto(eixoX: 7, eixoY: 4)
    

}

//teste
//var triangulo = Triangulo()
//print("Coordenada ds vértices do triângulo:")
//triangulo.vertice1.imprimeCoordenada()
//triangulo.vertice2.imprimeCoordenada()
//triangulo.vertice3.imprimeCoordenada()

